"Create today's package" step is broken for llvm-test-suite

We currently see this error when building the llvm-test-suite:

```
Something went wrong:
Error: Package llvm-test-suite already exists in copr @fedora-llvm-team/llvm-test-suite.
Error: Process completed with exit code 1.
```

This is because we re-use the `fedora-copr-build.yml` workflow in a way for which it wasn't intended before. It was written to support the forked-copr-project workflow. For the llvm-test-suite or for all `config.forked_repo == False` configurations we need to check if the package exits before we try to add/create it.

--- a/.github/workflows/fedora-copr-build.yml
+++ b/.github/workflows/fedora-copr-build.yml
@@ -164,10 +164,15 @@ jobs:
             copr-cli get-chroot "${{ env.project_today }}/$chroot"
           done
 
-      - name: "Create today's package"
+      - name: "Create/Edit today's package"
         if: ${{ env.skip_build_steps != 'yes' }}
         run: |
-          copr-cli add-package-scm \
+          copr_cmd="add-package-scm"
+          if copr-cli get-package --name ${{ matrix.copr_package_name }} "${{ env.project_today }}" > /dev/null 2>&1; then
+            echo "Package ${{ matrix.copr_package_name }} already exists in project ${{ env.project_today }}, editing package."
+            copr_cmd="edit-package-scm"
+          fi
+          copr-cli "${copr_cmd}" \
             --clone-url ${{ matrix.clone_url }} \
             --commit ${{ matrix.clone_ref }} \
             --spec ${{ matrix.spec_file }} \

